aws-auth-cognito/build.gradle.kts

```diff
@@ -54,12 +70,17 @@ dependencies {
     testImplementation(testDependency.androidx.test.core)
     testImplementation(testDependency.kotlin.reflection)
 
+
     androidTestImplementation(dependency.gson)
     //noinspection GradleDependency
     androidTestImplementation(testDependency.aws.sdk.core)
     androidTestImplementation(testDependency.androidx.test.runner)
     androidTestImplementation(testDependency.androidx.test.junit)
     androidTestImplementation(testDependency.kotlin.test.coroutines)
+
+    //androidTestImplementation("org.jetbrains.kotlinx:kotlinx-coroutines-android:+")
+
     androidTestImplementation(project(":aws-api"))
     androidTestImplementation(project(":testutils"))
+    implementation("org.jetbrains.kotlinx:kotlinx-coroutines-android:1.6.3")
```

Comment:
This was previously a test dependency only. Was it necessary to add to production code?